travvy/src/services/CovidService.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios'
+
+export default {
+  // It makes get request to recommend attractions based on city
+    getCovidInfo(country) {
+        if(country.country == 'England'){
+            const formated_country= 'UK'
+            const urlString = 'https://corona.lmao.ninja/v2/countries/'+ formated_country + '?yesterday&strict&query='
+            return axios.get(urlString)  
+            .catch(function (error) {
+              console.log(error);
+            });
+        } else if (country.country == 'United States'){
```

Comment:
It is really interesting to see how we are using a website to pull this information in the code. Great work!